tools/abel/README.md

```diff
@@ -0,0 +1,75 @@
+# ABEL
+#### Automated Building Entity Loader
+
+## Overview
+ABEL is a tool allowing system integrators to easily and efficiently create and modify [Building Configuration files](https://github.com/google/digitalbuildings/blob/master/ontology/docs/building_config.md).
+
+#### UDMI
+ABEL does not yet support [UDMI Site Model](https://github.com/faucetsdn/udmi/blob/master/docs/specs/site_model.md).
+
+## Setup and installation
+Similar to the [GUID Generator](https://github.com/google/digitalbuildings/tree/master/tools/guid_generator), ABEL's setup process depends on the instance validator. It is best to run the global setup script to setup the entire tools library which ABEL depends on
+
+1. `git clone https://github.com/google/digitalbuildings.git`
+2. `cd digitalBuildings/tools`
+3. Depending on your operating system run either of the global setup scripts to configure dependencies
+    * Windows: `pip_install.bat`
+    * Unix/MacOs: `sudo bash pip_install.sh`
+5. Create a directory for ABEL to use at `~/abel/` and navigate to it
+    * This will be used for storing authentication credentials and documents output from ABEL.
+4. Obtain a spreadsheet token for your google sheets account
+    1. Be added to a GCP service account with `Service Account User` and `Service Account Token Creator` roles.
+        * Please contact your IoT TPM for details and to be added to a project.
+    2. Create a token request file at `~/abel/token_request.json` with contents:
+        ```
+        {"scope": [    "https://www.googleapis.com/auth/spreadsheets" ],  "lifetime": "3600s"}
+        ```
+    3. Run the below curl command to obtain a token:
+        ```
+            curl -X POST \
+            -H "Authorization: Bearer "$(gcloud auth application-default print-access-token) \
+            -H "Content-Type: application/json; charset=utf-8" \
+            -d @token_request.json \
+            "https://iamcredentials.googleapis.com/v1/projects/-/serviceAccounts/your-service-account@project-id.iam.gserviceaccount.com:generateAccessToken" >| spreadsheet_token.json
+        ```
+    * `spreadsheet_token.json` should look something like:
+        ```
+        {
+          "accessToken": "ya29.c.AXv0zTOa9p5K78GtGntoIQCqyt2R572qZ3…",
+          "expireTime": "2022-02-24T19:57:26Z"
+        }
+        ```
+
+## The ABEL Spreadsheet
+The ABEL spreadsheet is what serves as a frontend for ABEL.
+
+Please see the [spreadsheet docs](./validators/README.md) for detailed documentation.
```

Comment:
use similar reference to avoid having the same bug as before related to github.io ...
https://github.com/google/digitalbuildings/commit/39c5776aec8aefb511609ed522a1f05f83cc9d3b

Reply:
Is this not a similar reference? I'm using a relative path just like that bug.